packages/launcher/src/renderer/launcher/apps/AppsView.js

```diff
@@ -115,13 +103,26 @@ type State = {
   },
   hover: ?string,
   showOnboarding: boolean,
+  suggestedApps: Array<Object>,
 }
 
 class AppsView extends Component<Props, State> {
   state = {
     hover: null,
     showModal: null,
     showOnboarding: false,
+    suggestedApps: [],
+  }
+
+  componentDidMount() {
+    this.fetchSuggested()
+  }
+
+  fetchSuggested = async () => {
+    const suggestedPromise = await fetch(SUGGESTED_APPS_URL)
```

Comment:
Probably best to wrap these calls in a try catch

Reply:
Oh yeah sure.. I actually forgot to do it..